infra/build/functions/requirements.txt

```diff
@@ -18,14 +18,14 @@ Brotli==1.0.9
 hiredis==1.1.0
 PyYaml==5.4
 PyGithub==1.51
-grpcio==1.29.0
-google-auth==1.21.1
-google-cloud-datastore<3.0
-google-cloud-ndb==1.7.1
-google-cloud-scheduler==1.3.0
-google-cloud-storage==1.29.0
-google-api-core==1.22.2
-google-api-python-client==1.9.3
+grpcio==1.47.0
```

Comment:
Why upgrade all of these? Can you get away with just upgrading one? I think it's riskier to do all.